### DISCUSSION TITLE
The page generator

### DISCUSSION THREAD

**Discussion by louiss0** [nxtensions/nxtensions#348] [General]
I have written the code that can be used for the new version of the page generator that exists. I hope that feature gets added soon 
It's in #31. Good Luck If you have any questions don't be afraid to ask.

**Comment by leosvelperez** [comment]
Hey @louiss0!

Thanks for your interest in getting this feature to land. Please note that there's no need to cross-post about it everywhere. Your upvotes and comments on the relevant issue are all that's needed. As mentioned in the issue, I'll work on it.
